.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           cache_key: v1-internal-test-app-{{ checksum "hydra-head.gemspec" }}-{{ checksum "spec/test_app_templates/lib/generators/test_app_generator.rb" }}-{{ checksum "hydra-core/lib/generators/hydra/head_generator.rb" }}-<< parameters.rails_version >>-<< parameters.ruby_version >>
 
       - samvera/install_solr_core:
-          solr_config_path: .internal_test_app/solr/config
+          solr_config_path: .internal_test_app/solr/conf
```

Comment:
Because `active-fedora` release 13.0.0 is not backwards-compatible with the latest changes, introducing this seems to break the builds for all previous `active-fedora` versions in the test build (please see https://github.com/samvera/hydra-head/blob/master/.circleci/config.yml#L103 and https://github.com/samvera/hydra-head/blob/master/.circleci/config.yml#L108).  I can remove these in order to get this passing, but I am uncertain if doing so breaks a level of committed supported for these older releases of `active-fedora`.

Reply:
could we set the path with an environment variable instead?